velox/docs/functions/spark/conversion.rst

```diff
@@ -277,4 +277,20 @@ Valid examples
   SELECT cast(cast(1.79769e+308 as double) as timestamp); -- 294247-01-10 04:00:54.775807
   SELECT cast(cast('inf' as double) as timestamp); -- NULL
   SELECT cast(cast('nan' as double) as timestamp); -- NULL
-  
\ No newline at end of file
+
+From boolean
+^^^^^^^^^^^^
+
+Casting from boolean to timestamp is supported.
+
+* ``true`` is cast to **1970-01-01 00:00:00.000001**
+* ``false`` is cast to **1970-01-01 00:00:00** (epoch)
+
+This behavior matches Spark's semantics for boolean-to-timestamp casts.
```

Comment:
Perhaps drop this sentence as this is the document for Spark casts, so the behaviours all match with Spark.